src/category_theory/limits/types.lean

```diff
@@ -104,6 +104,10 @@ begin
   simp [w j],
 end
 
+lemma limit_ext_iff (F : J ⥤ Type u) (x y : limit F) :
+  (∀ j, limit.π F j x = limit.π F j y) ↔ x = y :=
```

Comment:
`ext_iff` lemmas are usually written in reverse-simplifying order (so that you can use them in a `rw` chain without `<-`). Would you mind swapping the sides?